xtext/editorserver/spec/support/jasmine.json

```diff
@@ -0,0 +1,13 @@
+{
+  "spec_dir": "spec",
+  "spec_files": [
+    "**/*[sS]pec.?(m)js"
+  ],
+  "helpers": [
+    "helpers/**/*.?(m)js"
+  ],
+  "env": {
+    "stopSpecOnExpectationFailure": false,
```

Comment:
Why not?

Reply:
This ensures the test continues when there are multiple assertions. Can be useful for debugging as you can see more information in the test report. Overall the test will fail.